tools/install/Makefile.skel

```diff
@@ -16,16 +16,23 @@
 #
 # Makefile for AtChem2
 
-## paths to the dependencies
-## N.B.: use the full path, not the relative path
+# choose fortran compiler:
+# 1. "gnu" for gfortran
+# 2. "intel" for ifort
+FORTC = "gnu"
+
+# paths to the dependencies
+# N.B.: use the full path, not the relative path
 CVODELIB     = cvode/lib
 OPENLIBMDIR  = openlibm-0.4.1
 FRUITDIR     = fruit_3.4.3
 
-## default location of the chemical mechanism shared library (mechanism.so)
-## use the second argument of the build script to change $SHAREDLIBDIR
+# default location of the chemical mechanism shared library (mechanism.so)
+# use the second argument of the build script to change $SHAREDLIBDIR
```

Comment:
could you change this to "to override $SHAREDLIBDIR" as that's a bit more specific.